tests/tools/importexport/test_reader.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Tests for archive reader."""
+# pylint: disable=pointless-statement,redefined-outer-name
+import pytest
+
+from aiida.common import InvalidOperation
+from aiida.tools.importexport import EXPORT_VERSION, CorruptArchive
+from aiida.tools.importexport.archive import get_reader
+from tests.utils.archives import get_archive_file
+
+
+@pytest.fixture
+def archive_reader():
+    """Return an initiated archive reader."""
```

Comment:
a short documentation of example use would make sense here I think

Reply:
done